_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -0,0 +1,174 @@
+---
+title: 'Bitcoin Optech Newsletter #127'
+permalink: /en/newsletters/2020/12/09/
+name: 2020-12-09-newsletter
+slug: 2020-12-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Bech32 addresses for taproot and beyond:** Pieter Wuille
+  [published][wuille post] the results of research he conducted with
+  Gregory Maxwell into finding an optimal constant for updated [bech32
+  addresses][topic bech32] to both eliminate the [mutability problem][]
+  and mitigate other data entry problems as best as possible.  He also
+  replied to research performed by several individuals ([1][russell
+  test], [2][casatta test], [3][schmidt test1], [4][schmidt test2]) into
+  which wallets do and don't support [forward address compatibility][],
+  which revealed most wallets will need to be updated in order to pay
+  [taproot][topic taproot] addresses even if we continue using an
+  unchanged [BIP173][] address scheme.  Based on his research and the
+  other information available, and if there are no convincing
+  objections, he plans to write a BIP for the tweaked bech32 checksum
+  formula to be used for segwit v1 addresses (taproot) and subsequent
+  segwit versioned addresses.  If the proposed alternative bech32
+  addresses are adopted, the following implications exist for wallets
+  and services:
+
+    - *Support for current addresses remains unchanged:* wallets that
+      can send to current native segwit addresses (version 0), receive
+      to current segwit addresses, or do both will continue to work
```

Comment:
I think "or do both" is unnecessary:

"wallets that can send to current native segwit addresses (version 0) or receive to current segwit addresses will continue to work without modification."